CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## 23.1.2
 
+* Y011: Allow `math` constant `math.inf`, `math.e`, `math.pi`, `math.tau`, and
```

Comment:
V23.1.2 has already been released — this entry should go in a new "Unreleased" section immediately above the 23.1.2 section :)